docs/platform/howto/access-suspended-due.rst-to-past-due-invoice

```diff
@@ -0,0 +1,24 @@
+Access Suspended Due to Past Due Invoice
```

Comment:
```suggestion
Reactive suspended project
```

Reply:
I changed the title to be closer to what a user might search for if they get an error message saying "project suspended". It looks like the error message doesn't say why the project is suspended.

Reply:
`Reactivate` probably!

Reply:
Thanks @RyanSkraba. @isaevdd21 Could you fix this typo please?

Reply:
Thank you @RyanSkraba for catching this!